people/yumao/page.en.md

```diff
@@ -0,0 +1,152 @@
+Perhaps Yumao doesn't like taking part in group activities.
+I love white and red, as well as that kind of rainbow in Glitch.
+And I have a sweet tooth, I will feel happy for a long time while holding a packet of white rabbit creamy candies.
+I love cats, and adopt a cat called Ayaka.
+
+She was an admiral of *Kancolle* , and her first action figure was Destroyer Akizuki in *kancolle*.
+
+She loved time with silent and ease.
+For example, she put on the headphones and drank hot coffee in café, then watched out of the window and observed people came and left. This would make her relaxed.
+Sometimes, she had some special thoughts.
+She made wishes to meteors occasionally, she said: "It's not because I believe that, but in that it's free. And there's no evidence shows it doesn't work."
+
+According to Neko, Yumao loved spicy hot pot, and always enjoyed it with rice.
+
+Yumao liked to watch stars when she didn't sleep yet at night.
+After she returned to China, due to bad weather and few spare time, she couldn't watch stars as previous time.
+So [she made a special trip to Dishui Lake once and watched stars](https://web.archive.org/web/20210517104313/https://oao.moe/archives/834/).
+
+She saved many thoughts in her [private blog](https://web.archive.org/web/20210420170241/https://oao.moe/archives/), and these blog posts included many areas: computer science, military and politics. 
+She was also a book girl, and she wrote down much beautiful prose and poetry.
+Of course, she recorded daily life in her blog, and we can still see her story, her experience, and her train of thoughts now.
+
+## Experience
+
+Yumao once studied abroad in the US, then she backed to Shanghai to continue her study in university.
+
+In 2017, she encountered Neko in Twitter (Now X), and she felt that Neko was skilled and powerful.
+After that, they became close partners.
+
+In that period, there was not so much censorship about transgender topic in Zhihu.
+So Yumao always @ Neko in relevant topics, then showed their affection in public.
+
+During that time, Yumao got many mental supports. She described the care from Neko like this:
+
+> "Even those things were dark and painful, which had been hidden, you still observed and felt them. Maybe you understanded more then myself. How wonderful... You also found my scattered identity, then lifted me out of abyss. That was the first time I was been touched by others."
+
+However, they were forced to separate.
+After that, Neko was brought back to Yunnan by family, and was sent to correctional school.
+Then, it was reported that Neko had already died by suicide.
+Yumao was extremely grieved after hearing the news.
+She continuously sent messages about missing Neko in their Telegram channel.
+She quoted *Cassiopeia(カシオペヤ)* , which was a song they used to listen together, to express her grief.
+
+Although she also found that the suicide of Neko was complicated and confusing according to some routes.
```

Comment:
Although she also found that the circumstances of the suicide of Neko was complicated and confusing according to some sources.